_posts/2025-09-30-add-new-table-to-capture-list.adoc

```diff
@@ -0,0 +1,136 @@
+date:   2025-09-30
+tags: [ cdc, integration, configuration, howto ]
+author: mfvitale
+---
+
+"How can I add a new table to start capturing its changes?"
+
+This is one of the most common questions from our community.
+
+If you ask The Ultimate Hitchhiker’s Guide to the Galaxy, the answer to everything is 42.
+Sadly, in the real world, it’s not quite that simple. The real answer is: it depends.
+
+With this article, I’d like not only to provide an answer by walking through different scenarios,
+but also to explain the reasons behind it.
+
++++<!-- more -->+++
+
+== Introduction
+Before exploring all the possibilities, we first need to define some basic Debezium concepts.
+
+_How does Debezium know which tables to capture?_
+
+Debezium offers configurations to set which database/schema/table to capture via the following properties:
+
+`database.include.list`:: An optional, comma-separated list of regular expressions that match the names of the databases from which the connector captures changes. The connector does not capture changes in any database whose name is not in database.include.list. By default, the connector captures changes in all databases.
+`schema.include.list`:: An optional, comma-separated list of regular expressions that match names of schemas for which you want to capture changes. Any schema name not included in schema.include.list is excluded from having its changes captured. By default, all non-system schemas have their changes captured.
+`table.include.list`:: An optional, comma-separated list of regular expressions that match fully-qualified table identifiers for tables whose changes you want to capture. When this property is set, the connector captures changes only from the specified tables. Each identifier is of the form schemaName.tableName. By default, the connector captures changes in every non-system table in each schema whose changes are being captured.
+
+there are also properties to apply opposite logic:
+
+`database.exclude.list`:: An optional, comma-separated list of regular expressions that match the names of databases from which you do not want the connector to capture changes.
+`schema.exclude.list`:: An optional, comma-separated list of regular expressions that match names of schemas for which you do not want to capture changes.
+`table.exclude.list`:: An optional, comma-separated list of regular expressions that match fully-qualified table identifiers for tables whose changes you do not want to capture.
+
+the final configuration of these properties tells Debezium which tables to capture events from.
+
+_How Debezium gets table schema information?_
+
+Debezium, in order to start streaming data changes from a table, needs to know its schema so it can provide information about the table’s structure.
+Normally, Debezium learns about table schemas while streaming by reading DDL statements, parsing them, and updating an in-memory representation of each table’s schema.
```

Comment:
I think this part needs an expanison. In fact there are mutliple options here
DDL - e.g. MariaDB, MySQL, Oracle
metadata message - e.g. PostgreSQL
JDBC - e.g. SQL Server, Db2